CONTRIBUTING.md

```diff
@@ -6,12 +6,6 @@ Welcome to [TheAlgorithms/Go](https://github.com/TheAlgorithms/Go)! Before submi
 
 ## Contributing
 
-### Maintainer/reviewer
-
-**Please check the [reviewer code](https://github.com/TheAlgorithms/Go/blob/master/DIRECTORY.md) file for maintainers and reviewers.**
```

Comment:
Instead of removing this section, point it to the [`CODEOWNERS`](https://github.com/TheAlgorithms/Go/blob/master/.github/CODEOWNERS) file.